x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/action/InternalInferModelActionRequestTests.java

```diff
@@ -66,23 +70,4 @@ protected NamedWriteableRegistry getNamedWriteableRegistry() {
         entries.addAll(new MlInferenceNamedXContentProvider().getNamedWriteables());
         return new NamedWriteableRegistry(entries);
     }
-
-    @Override
-    protected Request mutateInstanceForVersion(Request instance, Version version) {
-        if (version.before(Version.V_7_8_0)) {
```

Comment:
Since the 7.9 branch was cut this code won't be hit from the master branch. I will make the adjustments on the backport

Reply:
We should keep this as `AbstractBWCWireSerializationTestCase` and just remove these mutate branches.